addons/mail/models/discuss/res_partner.py

```diff
@@ -18,6 +18,13 @@ class ResPartner(models.Model):
         copy=False,
     )
     channel_member_ids = fields.One2many("discuss.channel.member", "partner_id")
+    is_in_call = fields.Boolean(compute="_compute_is_in_call")
+    rtc_session_ids = fields.One2many(comodel_name="discuss.channel.rtc.session", inverse_name="partner_id")
```

Comment:
Both probably need a `groups` on them to avoid either access error and/or "leaking" data.

Reply:
The comment was mainly meant for the `is_in_call` field, as it is defined on partner with sudo, it means the call status of everyone become accessible to everyone else, which we wanted to avoid. Info should only be returned as sudo in the livechat flow instead.